test/test_nn.py

```diff
@@ -4076,32 +4076,35 @@ def test_pruning_container_compute_mask(self):
         default_mask = torch.tensor([[1, 1, 1, 0], [1, 1, 0, 1]])
         # since we are pruning the two lowest magnitude units, the outcome of
         # the calculation should be this:
-        expected_mask = torch.tensor([[0, 0, 1, 0], [1, 1, 0, 1]])
+        expected_mask = torch.tensor([[0, 0, 1, 0], [1, 1, 0, 1]]).to(dtype=torch.float32)
```

Comment:
Pass `dtype` directly as a parameter to `torch.tensor` instead of using `to`, which creates a copy. Here and everywhere.